optuna/visualization/_parallel_coordinate.py

```diff
@@ -31,6 +33,23 @@
 _logger = get_logger(__name__)
 
 
+class _DimensionInfo(NamedTuple):
+    label: str
+    values: Tuple[float, ...]
+    range: Tuple[float, float]
+    is_log: bool = False
+    is_cat: bool = False
+    tickvals: List[int] = []
+    ticktext: List[str] = []
```

Comment:
One more thing. What do you think about https://github.com/optuna/optuna/pull/3781#discussion_r918605645?

Reply:
Thanks, fixed!